filters/ThirdParty/filter_255_EasyListPortuguese/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filterId": 255,
+  "name": "EasyList Portuguese",
+  "description": "Additional filter list for websites in Spanish, Portuguese.",
```

Comment:
`Spanish, Portuguese` maybe better `and` instead of `,`? (also in another files)

Reply:
I guess there is no difference in the meaning, but there is only 2 languages, so we can use `and` instead of the comma.